internal/core/client/resolver.go

```diff
@@ -34,7 +35,7 @@ func (client *Client) ResolvePath(ctx context.Context, raw string) (*types.Resol
 
 	res.RepoName = parts[1]
 	res.Repository, err = client.GetRepository(ctx, orgID, res.RepoName)
-	if err != nil {
+	if err != nil && !errors.Is(err, types.ErrRepositoryNotExist) {
```

Comment:
I think the `NotExist` errors should be returned.  Continuing to resolve after a missing path doesn't make sense.

Reply:
fixed this by returning &res, err, allowing for error checking as well as using anything that did get resolved